tortoise.py

```diff
@@ -2,19 +2,36 @@
 
 
 def main():
-    
+    VALID_ARGS = ['tiny.tortoise', 'medium.tortoise', 'large.tortoise']
+
+    FILENAME = get_tortoise_file(sys.argv, VALID_ARGS)
+    if not '.tortoise' in FILENAME:
+        print(FILENAME)
+        exit()
+
+    content = get_file_content(FILENAME)
+    for i in content:
+        print(i, end='\n')
+
+
+def get_tortoise_file(argv, VALID_ARGS):
```

Comment:
If this function returns a string filename on success, it's dangerous for it to return a string error message on failure. How could the calling code tell the difference? Is there a better way?